### ISSUE TITLE
Plain JavaScript experience

### ISSUE THREAD

**Issue by raymondfeng** [strongloop/loopback-next#560]
As a developer that uses plain JavaScript, I would like to use LoopBack Next without requiring TypeScript specific features such as decorators to:

1. Implement controllers/repositories/services
2. Create extensions 

**UPDATED (2/22)**
Discussion with @raymondfeng @kjdelisle:
- need to go through what is supported. 
- if it is too aggressive to support JavaScript right away, we can defer from Core-GA.  
- bottom line: need to let the users know what is supported.
- a spike is needed

**Comment by ansarizafar** [comment]
@loopback/cli creates a typescript project. Is there any javascript starter available?

**Comment by bajtos** [comment]
@ansarizafar not yet. We are focusing on getting a typescript-only MVP done first (see https://github.com/strongloop/loopback-next/issues/696). We we look into JavaScript support after this MVP is done.
